eng/pipelines/runtime-official.yml

```diff
@@ -0,0 +1,98 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - release/*
+
+pr: none
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+# TODO: (Consolidation) Switch away from old signing/validation variables from former Core-Setup.
+- name: TeamName
+  value: dotnet-core-acquisition
```

Comment:
Why is that the TeamName? This should just be .NET Core, no? Or .NET Runtime?

Reply:
Yep, this is what the TODO is about, `TODO: (Consolidation) Switch away from old signing/validation variables from former Core-Setup.`

Reply:
Ok, cool. As long all these are tracked by an issue.